benchmark/utils/CxxTests/GlobalVars.h

```diff
@@ -0,0 +1,16 @@
+static int globalInt = 42;
+static constexpr int globalConstexprInt = 42;
+static float globalFloat = 42;
+static constexpr float globalConstexprFloat = 42;
+
+struct BigObject {
+  char buff[128];
```

Comment:
Is there a reason for 128 bytes?  Id rather this be larger (512 feels about right).

Reply:
No reason. I'll change it.